apps/web/components/admin/audit-log-viewer.tsx

```
@@ -0,0 +1,159 @@
+
+          {data?.total > filters.limit && (
+            <div className="flex justify-between items-center">
+              <div className="text-sm text-gray-600">
+                Showing {filters.offset + 1} to {Math.min(filters.offset + filters.limit, data.total)} of {data.total} entries
+              </div>
+              <div className="flex gap-2">
+                <button
+                  onClick={() => handlePageChange(Math.max(0, filters.offset - filters.limit))}
+                  disabled={filters.offset === 0}
+                  className="px-3 py-1 border rounded disabled:opacity-50"
+                >
```

Biome (2.4.9)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)